app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -18,14 +21,28 @@ public ProductService(ProductRepository productRepository) {
         this.productRepository = productRepository;
     }
 
+    /**
+     * 등록된 모든 상품의 목록을 리턴합니다.
+     * @return 상품목록
+     */
     public List<Product> getProducts() {
         return productRepository.findAll();
     }
 
+    /**
+     * 식별 값으로 상품을 조회하고, 조회 된 상품을 리턴합니다.
+     * @param id 식별 값
+     * @return 조회 된 상품
+     */
```

Comment:
이 정도로도 충분하겠지요. 이제 javadoc은 굉장히 잘 작성하시는데요, 간결하게만 적으실 수 있으면 훨씬 문장이 좋아질 거에요.
```suggestion
    /**
     * 상품을 조회해 리턴합니다.
     * @param id 상품 식별값
     * @return 조회된 상품
     */
```

Reply:
수정해주신걸 보면 간결하고 의미가 잘 전달되는데
막상 제가 간결하게 하려고 하면 생략해도 될까? 걱정되네요 꾸준히 작성해보겠습니다 ~

Reply:
ㅎㅎ 무조건 많이 해보세요. 하다보면 점점 나아질 거에요.